File: app/src/main/java/in/kashewdevelopers/xo/GameRoom.java
package in.kashewdevelopers.xo;

import androidx.annotation.NonNull;

public class GameRoom {

    public Boolean creatorChance, gameFinished, newGame, leftGame;
    public String creatorName, joineeName;
    public Integer movePlayed;

    GameRoom() {
    }

    GameRoom(Boolean creatorChance, String creatorName, String joineeName, Integer movePlayed) {
        this.creatorChance = creatorChance;
        this.creatorName = creatorName;
        this.joineeName = joineeName;
        this.movePlayed = movePlayed;
        this.leftGame = false;
    }

    @NonNull
    @Override
    public String toString() {
        return "creatorChance : " + creatorChance +
                ", creatorName : " + creatorName +
                ", joineeName : " + joineeName +
                ", newGame : " + newGame +
                ", gameFinished : " + gameFinished +
                ", leftGame : " + leftGame +
                ", movePlayed : " + movePlayed;
    }
}

File: app/src/main/java/in/kashewdevelopers/xo/GameRoom.kt
package `in`.kashewdevelopers.xo

data class GameRoom(var creatorChance: Boolean,
                    var creatorName: String,
                    var joineeName: String,
                    var movePlayed: Int) {

    var leftGame: Boolean = false
    var newGame: Boolean = false
    var gameFinished: Boolean = false

    override fun toString(): String {
        return "creatorChance : $creatorChance, " +
                "creatorName : $creatorName, " +
                "joineeName : $joineeName, " +
                "newGame : $newGame, " +
                "gameFinished : $gameFinished, " +
                "leftGame : $leftGame, " +
                "movePlayed : $movePlayed"
    }

}